src/main/java/com/minecolonies/coremod/commands/GetColonyAndCitizen.java

```diff
@@ -91,18 +91,18 @@ else if (args.length >= TOO_MANY_ARGUMENTS_LENGTH)
      * @param args     The given arguments
      * @return Returns citizen ID
      */
-    public static int getCitizenId(@NotNull int colonyId, @NotNull final String... args)
+    public static int getCitizenId(@NotNull final int colonyId, @NotNull final String... args)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "getCitizenId" is 16 which is greater than 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 25 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)